inst/schema/spec.md

```diff
@@ -0,0 +1,229 @@
+This is all highly uncertain and liable to change!
+
+# Data
+## POST /data/tab-name/
+Accepts a set of baseline options, and returns a data set for the data under the named tab.
+
+Request schema: [DataOptions.schema.json](./DataOptions.schema.json)
+
+Return schema: [Data.schema.json](./Data.schema.json)
+
+### Example
+#### Request
+```
+{
+  "irs_future": "80%",
```

Comment:
Can we take percentages as either 0.8 or 80? or is this in order to match what is effectively a categorical variable in the lookup?

Reply:
at the moment yes it's a categorical variable, but no reason to pass it in string form I don't think